packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -477,6 +478,25 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
     logRecoverableError() {
       // no-op
     },
+
+    getCurrentEventStartTime(): number {
+      return Scheduler.unstable_now();
+    },
+
+    scheduleTransitionCallbacks(
+      callback: (
+        Array<TransitionCallbackObject>,
+        endTime: number,
+        callbacks: TransitionTracingCallbacks,
+      ) => void,
+      pendingTransitions: Array<TransitionCallbackObject>,
+      callbacks: TransitionTracingCallbacks,
+    ): void {
+      const endTime = Scheduler.unstable_now();
+      Scheduler.unstable_scheduleCallback(Scheduler.unstable_IdlePriority, () =>
```

Comment:
If the only thing this does is call Scheduler, we might as well call Scheduler directly from the reconciler instead of going through the host config. We can add a host config once it becomes necessary.

Same for `getCurrentEventStartTime`, I think. Just call `Scheduler.unstable_now` for now.